oci/spec_opts.go

```diff
@@ -333,7 +333,13 @@ func WithImageConfigArgs(image Image, args []string) SpecOpts {
 			return WithAdditionalGIDs("root")(ctx, client, c, s)
 		} else if s.Windows != nil {
 			s.Process.Env = config.Env
-			s.Process.Args = append(config.Entrypoint, config.Cmd...)
+			cmd := config.Cmd
+			if len(args) > 0 {
+				cmd = args
+			}
+			s.Process.Args = append(config.Entrypoint, cmd...)
+
+			s.Process.Cwd = config.WorkingDir
```

Comment:
Should there be a test for `config.WorkingDir == ""` to default it to `C:\` for Windows as it's done from Linux above in L321?

Reply:
What do you need to change the existing code? Seems like all that is new is the `s.Process.Cwd = config.WorkingDir`. Doesn't `config.Cmd...` expand properly to zero automatically by golang?

Reply:
Seems the model is inconsistent already. We should likely be using `replaceOrAppendEnvValues` everywhere. But at a minimum I think yes we should be appending here too otherwise call order would really matter for env's.

Reply:
I added following changes:
WithImageConfigArgs will override env,args,cwd of all previous WithXxx calls.
env will be merged using replaceOrAppendEnvValues instead of append.
args,cwd  will be just replaced